flask_caching/__init__.py

```diff
@@ -733,7 +745,7 @@ def _memoize_kwargs_to_args(self, f, *args, **kwargs):
         new_args.extend(args[len(arg_names) :])
         return (
             tuple(new_args),
-            OrderedDict(
+            dict(
```

Comment:
`OrderedDict`was used to keep the dictionary, well, ordered. If you think ordering is not needed here, please explain better in the PR description. If it *is* not needed, the `sorted` operation should also be removed in the following line.

Reply:
thanks for your reply.
python3.6 later，dict has the order，and used less memory。